client/extensions/iptc/src/extension.ts

```diff
@@ -0,0 +1,61 @@
+import {IExtension, IExtensionActivationResult, ISuperdesk, IArticle, ISubject} from 'superdesk-api';
+
+type Scheme = 'media-credit' | 'belga-keywords' | 'countries';
+
+const CVS: Array<Scheme> = ['media-credit', 'belga-keywords', 'countries'];
+
+const DEFAULT_SUBJECT = {
+    'media-credit': ['BELGA_ON_THE_SPOT'],
+    'belga-keywords': ['RUSHES', 'BELGAILLUSTRATION', 'BELGAINTERVIEW', 'BELGAINSERT'],
+    'countries': ['BEL'],
```

Comment:
If you need a single selection, you can use a `country` CV https://github.com/superdesk/superdesk-belga/blob/master/server/data/vocabularies.json#L4204